drape/overlay_tree.cpp

```diff
@@ -395,7 +395,7 @@ void OverlayTree::Select(m2::PointD const & glbPoint, TOverlayContainer & result
 
   for (auto const & handle : m_handlesCache)
   {
-    if (rect.IsPointInside(handle->GetPivot(screen, false)))
+    if (!handle->HasDynamicAttributes() && rect.IsPointInside(handle->GetPivot(screen, false)))
```

Comment:
Yes, it’s ok. Select(by Point) is used for finding z coordinate at that point and here we need to check all the overlays.